### PR TITLE
PR for Kata1 tests

### PR DESCRIPTION
**Pull request by deanosaurx**
Finished the last test.

### CODE REVIEW

File: python_katas/kata_1/test.py [hunk 1]
```diff
@@ -64,7 +88,17 @@ class TestIsUniqueString(unittest.TestCase):
...
+
+        testUnique = "aa"
+        self.assertEqual(questions.is_unique_string(testUnique), False)
+
+        testUnique = "ENJOY THIS-PARK"
+        self.assertEqual(questions.is_unique_string(testUnique), True)
+
+        testUnique = "asdfghjklzxcvbnma"
+        self.assertEqual(questions.is_unique_string(testUnique), False)
         pass
```
Comment (ronhadad22) [inline python_katas/kata_1/test.py:102]: what about empty str

File: python_katas/kata_1/test.py [hunk 2]
```diff
@@ -74,65 +108,172 @@ class TestListDiff(unittest.TestCase):
...
-        # your code here
+        listToDiff = [1, 3, 6, 10, 14, 25]
+        self.assertEqual(questions.list_diff(listToDiff), [2, 3, 4, 4, 11])
+
+        listToDiff = [4, 1, 7, 2, 11, 0]
+        self.assertEqual(questions.list_diff(listToDiff), [-3, 6, -5, 9, -11])
+
+        listToDiff = [5000, 1, 10000, 10]
+        self.assertEqual(questions.list_diff(listToDiff), [-4999, 9999, -9990])
         pass
```
Comment (ronhadad22) [inline python_katas/kata_1/test.py:119]: the first element is NULL

File: python_katas/kata_1/test.py [hunk 3]
```diff
@@ -74,65 +108,172 @@ class TestListDiff(unittest.TestCase):
...
+        self.assertTrue(questions.palindrome_num(num), "the num is palindrome but you return False")
+
+    def test_pal1False(self):
+        num = 113
+        self.assertFalse(questions.palindrome_num(num),"the num is not palindrome but you return True")
+
+    def test_pal1Zero(self):
+        num = 0
+        self.assertTrue(questions.palindrome_num(num)," 0 is palindrom num but you return False")
+
```
Comment (ronhadad22) [inline python_katas/kata_1/test.py:173]: func need to return bool

File: python_katas/kata_1/test.py [hunk 4]
```diff
@@ -141,36 +282,76 @@ class TestMergeDicts(unittest.TestCase):
...
+        # Test a number that is a multiple of 7
+        self.assertEqual(seven_boom(49), ["Boom!"])
+
+        # Test a number that is a multiple of 7 and has a 7 in it
+        self.assertEqual(seven_boom(77), ["Boom!", "Boom!"])
+
+        # Test a number that is a multiple of 7 and has a 7 in it and ends in 7
+        self.assertEqual(seven_boom(707), ["Boom!", "Boom!", "Boom!"])
+
```
Comment (ronhadad22) [inline python_katas/kata_1/test.py:320]: the return value is a list!

File: python_katas/kata_1/test.py [hunk 5]
```diff
@@ -141,36 +282,76 @@ class TestMergeDicts(unittest.TestCase):
...
+        # Test a string with only uppercase letters
+        self.assertEqual(caesar_cipher("HELLO", 3), "KHOOR")
+
+        # Test a string with both lowercase and uppercase letters
+        self.assertEqual(caesar_cipher("Hello, World!", 3), "Khoor, Zruog!")
+
+        # Test a string with a shift value of 0
+        self.assertEqual(caesar_cipher("Hello, World!", 0), "Hello, World!")
+
+
```
Comment (ronhadad22) [inline python_katas/kata_1/test.py:340]: the input is str!

File: python_katas/kata_1/test.py [hunk 6]
```diff
@@ -141,36 +282,76 @@ class TestMergeDicts(unittest.TestCase):
...
-        pass
+    def test_sum_of_digits(self):
+        # Test a string with only digits
+        self.assertEqual(sum_of_digits("12345"), 15)
+
+        # Test a string with digits and non-digit characters
+        self.assertEqual(sum_of_digits("a1b2c3d4e5"), 15)
+
+        # Test an empty string
+        self.assertEqual(sum_of_digits(""), 0)
```
Comment (ronhadad22) [inline python_katas/kata_1/test.py:355]: from the function description, "you can assume the input is a string containing numeric digits only"

**Review by ronhadad22 — CHANGES_REQUESTED** [review]
pls FIX all  the sections that with comments

File: python_katas/kata_1/test.py (hunk 1 above)
Comment (deanosaurx) [reply python_katas/kata_1/test.py]: Added empty string with the latest commit

File: python_katas/kata_1/test.py (hunk 3 above)
Comment (bibiefart) [reply python_katas/kata_1/test.py:171]: fixed
instaead of assertTrue/assertFalse i used assertEqual

File: python_katas/kata_1/test.py (hunk 4 above)
Comment (deanosaurx) [reply python_katas/kata_1/test.py:331]: Fixed by Dean.

File: python_katas/kata_1/test.py (hunk 5 above)
Comment (deanosaurx) [reply python_katas/kata_1/test.py:351]: Fixed by Dean.

**Review by ronhadad22 — APPROVED** [review]

**Pull request merged**